lib/encoding/bufferreader.js

```diff
@@ -91,11 +91,16 @@ BufferReader.prototype.readUInt64BEBN = function() {
 };
 
 BufferReader.prototype.readUInt64LEBN = function() {
-  var buf = this.buf.slice(this.pos, this.pos + 8);
-  var reversebuf = BufferReader({
-    buf: buf
-  }).readReverse();
-  var bn = BN.fromBuffer(reversebuf);
+  var second = this.buf.readUInt32LE(this.pos);
+  var first = this.buf.readUInt32LE(this.pos + 4);
+  var combined = (first * 0x100000000) + second;
```

Comment:
It can lose precision with numbers equal or above `2 ^ 53`.

```js
> 0x1fffffffffffff === Math.pow(2, 53) - 1
// true
```
```js
> 1152921504606847000 < 0x1fffffffffffff
// false
```